### PR TITLE
feat(campaign): fix ui

### PR DESCRIPTION
**Pull request by nguyenhuudungz**

### CODE REVIEW

File: src/pages/Campaign/index.tsx [hunk 1]
```diff
@@ -402,7 +402,9 @@ export default function Campaign() {
                         ? getFormattedTimeFromSecond((selectedCampaign.startTime - now) / 1000)
                         : selectedCampaign.status === 'Ongoing'
                         ? getFormattedTimeFromSecond((selectedCampaign.endTime - now) / 1000)
-                        : 'ENDED'
+                        : new Date(selectedCampaign.endTime).toISOString().slice(0, 10) +
```
Comment (nguyenhoaidanh) [inline src/pages/Campaign/index.tsx:405]: should left a comment.
```
// '2022-07-28 11:45' 
// 'yyyy-mm-dd hh:mm' 
```
anyway, we have dayjs can format for us.

Comment (nguyenhuudungz) [reply src/pages/Campaign/index.tsx:405]: @nguyenhoaidanh fixed.

Comment (nguyenhoaidanh) [reply src/pages/Campaign/index.tsx:405]: Have you committed?

Comment (nguyenhuudungz) [reply src/pages/Campaign/index.tsx]: Yeah.

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**